docs/content/en/docs/osmgmt/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Overview"
+linkTitle: "Overview"
+weight: 5
+description: >
+  Overview of operating system management for nodes in EKS Anywhere clusters.
+---
+
+Bottlerocket, Ubuntu, and Red Hat Enterprise Linux (RHEL) can be used as operating systems for nodes in EKS Anywhere clusters. You can only use a single operating system per cluster. Bottlerocket is the only operating system distributed and fully supported by AWS. If you are using the other operating systems, you must build the operating system images and configure EKS Anywhere to use the images you built when installing or updating clusters. AWS will assist with troubleshooting and configuration guidance for Ubuntu and RHEL as part of EKS Anywhere Enterprise Subscriptions. For official support for Ubuntu and RHEL operating systems, you must purchase support through their respective vendors.
+
+Reference the table below for the operating systems supported per deployment option for the latest version of EKS Anywhere (v0.16.0). See [Admin machine]({{<  relref "/docs/getting-started/install" >}}) for supported operating systems.
+
+|| vSphere | Bare metal | Snow | CloudStack | Nutanix |
+| --- | :---: | :---: | :---: | :---: | :---: |
+| Bottlerocket | &#10004; | &#10004; | &mdash; | &mdash; | &mdash; |
+| Ubuntu | &#10004; | &#10004; | &#10004; | &mdash; | &#10004; |
+| RHEL | &#10004; | &#10004; | &mdash; | &#10004; | &mdash; |
+
+| OS | Supported Versions |
+| :---: | :---: |
+| Bottlerocket | 1.13.x |
+| Ubuntu | 20.04.x |
+| RHEL | 8.x |
+
+With the vSphere, bare metal, Snow, CloudStack and Nutanix deployment options, EKS Anywhere provisions the operating system when new machines are deployed during cluster creation, upgrade, and scaling operations. You can configure the operating system to use through the EKS Anywhere cluster spec, which varies by deployment option. See the deployment option sections below for an overview of how the operating system configuration works per deployment option.
+
+## vSphere
+To configure the operating system to use for EKS Anywhere clusters on vSphere, use the `VSphereMachineConfig` template field. The template name corresponds to the template you imported into your vSphere environment. See the Customize OVAs and Import OVAs documentation pages for more information. Changing the template after cluster creation will result in the deployment of new machines.
+
```

Comment:
Bit of a macro question here, but should we mention that users could also run a `apt-get update` to upgrade their OS version?